requirements.txt

```diff
@@ -1,4 +1,4 @@
-pynwb
+pynwb>=2.8  # NWB Inspector should always be used with most recent minor versions of PyNWB
```

Comment:
Will this cause any difficulties when trying to generate example files with particular failures as in: https://github.com/NeurodataWithoutBorders/nwbinspector/blob/4856bd65fa0650080b1ecb7f225596be6962208d/src/nwbinspector/testing/_testing.py#L90

maybe there should be a separate `requirements-dev.txt` with no minimum pynwb version specified?

Reply:
When using that specific method, you can always do

```bash
pip install nwbinspector
pip install pynwb==2.1.0
```

as separate steps to obtain the desired result

Towards #507 there should be no excessive requirements files

Reply:
Following those `pip install` steps leads to some dependency conflict warnings, though the tests seem to run fine so that works.

Reply:
@stephprince Exactly, it's a warning rather than an error (if you do `pip install nwbinspector pynwb==2.1.0` or include `pynwb==2.1.0` in the requirements, it outright fails) but therefore lets you override it to 'force' the incompatability